core/commands/bootstrap.go

```diff
@@ -236,7 +239,18 @@ var bootstrapRemoveCmd = &cmds.Command{
 	Type: BootstrapOutput{},
 	Marshalers: cmds.MarshalerMap{
 		cmds.Text: func(res cmds.Response) (io.Reader, error) {
-			v, ok := res.Output().(*BootstrapOutput)
+			var (
+				ch <-chan interface{}
+				ok bool
+			)
+
+			if ch, ok = res.Output().(<-chan interface{}); !ok {
+				if ch, ok = res.Output().(chan interface{}); !ok {
```

Comment:
Hmm I think I tried that due to the weird shimming layer, but I'll take another look